gcs/holder.py

```diff
@@ -14,16 +14,15 @@
 
 """Implement a holder for resumable upload's data and rewrite's data"""
 
-from typing import ClassVar
-import crc32c
 import flask
```

Comment:
Thanks for the cleanup! `flask` should be moved down?

Reply:
Fixed.